Makefile

```diff
@@ -12,6 +12,7 @@ RELEASE := 1
 OFFICIAL := yes
 APPVER := $(VERSION)
 endif
+SKIP_KUBE=0
```

Comment:
The way this is written breaks the ability to override this variable via an environment variable. Let's use this stripped-down Makefile as an example:

```make
SKIP_KUBE=0

test:
	@echo $(SKIP_KUBE)
```

The output is:

```
$ SKIP_KUBE=1 make test
0
```

If written it like this:


```make
SKIP_KUBE ?= 0

test:
	@echo $(SKIP_KUBE)
```

It works as expected:

```
$ SKIP_KUBE=1 make test
1
```